### ISSUE TITLE
Background image not covering whole page

### ISSUE THREAD

**Issue by mladenabaz** [keycloakify/keycloakify#586]
Hi @garronej, 

thanks for the keycloakify, it's been really helpful to me for building my first custom keycloak theme.

As title states, I'm having some issues with the backgorund image on my theme.  As you can see on my screenshots my background image looks ok, meaning the background is covering the whole page on one window size but if the height is such that I have overflow it's not covering the whole page.

I'm not sure if this is even an issue related to keycloakify but I get the impression that you could help me anyway :) I would highly appreciate it.

I addedd background image via css::
body.kcBodyClass {
    background-image: url(./assets/iddeea-background.webp);
    background-position: center center;
    background-size: cover;
}

Thanks a lot!

![Screenshot 2024-07-11 at 21 41 46](https://github.com/user-attachments/assets/481a4132-8190-496b-8ff6-0259d1493a02)
![Screenshot 2024-07-11 at 21 42 04](https://github.com/user-attachments/assets/e6416619-7428-4e8d-9fbe-33335efd64f9)

**Comment by garronej** [comment]
Hi @mladenabaz,

Thanks for reaching out and I'm glad to hear that keycloakify has been helpful for your project.  



Try:  

```css
body.kcBodyClass {
    background-image: url(./assets/iddeea-background.webp);
    background-position: center center;
    background-size: cover;
    background-repeat: no-repeat;
    min-height: 100vh; /* Ensure the background covers at least the full viewport height */
    overflow: auto; /* Allow scrolling if content overflows */
    display: flex;
    flex-direction: column;
}
```

**Comment by mladenabaz** [comment]
Thanks for swift response.

This css didn't help after all but I've sorted it finally with these changes. Not sure if this is the proper way to accomplish it but it works :)

After further css inspection I figured there are some patternfly classes setting background image and background color so I tried to changing those and managed to get this working.


```
.login-pf {
    background: unset;
}

.login-pf body {
    background: url(./assets/iddeea-background.webp) no-repeat center center fixed;
    background-size: cover;
}
```
